curriculum/challenges/english/09-information-security/information-security-projects/anonymous-message-board.english.md

```diff
@@ -23,27 +23,44 @@ Start this project on Repl.it using <a href="https://repl.it/github/freeCodeCamp
 
 ```yml
 tests:
+  - text: I can provide my own project, not the example URL.
+    testString: "getUserInput => {
+      const url = getUserInput('url');
+      assert(!(new RegExp('.*/anonymous-message-board--freecodecamp\\.repl\\.co')).test(getUserInput('url')));
```

Comment:
Interested as to why not use the `const url` here?

Reply:
Haha whoops! Thanks for the catch!

Wanted to pass that value down the line of tests, but it looks like the tests are independently scoped. Forgot to clean up the stranded declaration. 😅